mm/slab.h

```diff
@@ -324,7 +325,7 @@ static inline void memcg_slab_post_alloc_hook(struct kmem_cache *s,
 
 	flags &= ~__GFP_ACCOUNT;
 	for (i = 0; i < size; i++) {
-		if (likely(p[i])) {
+		if (likely(p[i] && !is_kfence_address(p[i]))) {
```

Comment:
We probably need a comment here about why KFENCE objects do not need to be charged.

Reply:
This commit is being reverted. :-) 

I just left it here for keeping the history.